src/scripts/quiz_controller.js

```diff
@@ -0,0 +1,54 @@
+import { Controller } from 'stimulus-repo/packages/stimulus'
+
+export class QuizController extends Controller {
+  static targets = ['questions', 'submit', 'results', 'restart']
+  static parent = 'math'
+  static children = ['equation']
```

Comment:
As part of using webpack, the javascript is transpiled, meaning we can use new features that browsers might not now about and the code will be rewritten so that it works in browsers. This code used to have a `get` and curly braces with a `return`. This is the new static property feature, which is easier to read; it gets transpiled back to the old way.